pkg/conversion/conversion.go

```diff
@@ -0,0 +1,32 @@
+package conversion
+
+import "fmt"
+
+const (
+	MinInt32  = -(1 << 32)
+	MaxInt32  = 1<<31 - 1
+	MinUint   = 0
+	MinUint32 = 0
+	MaxUint32 = 1<<32 - 1
```

Comment:
the built-in [`math` package](https://pkg.go.dev/math#pkg-constants) provides a set of max/min consts including `MaxUint`.